### ISSUE TITLE
Add more login providers

### ISSUE THREAD

**Issue by Friedenspanzer** [Friedenspanzer/meeple-party#82]
The providers currently implemented were the easiest ones but real users will surely demand others.

Must-have providers:

- ~~Twitter~~
- ~~Facebook~~

Nice-to-have providers:

- ~~Reddit~~
- [ ] Mastodon (Currently not supported by Authjs)
- [ ] Apple
- [x] Discord
- [x] Twitch

It would be great to allow people to login directly through BGG. But as far as I know they don't support OAuth or any other technique. The GitHub login is not strictly necessary and may be removed when there's to many buttons.

**Comment by Friedenspanzer** [comment]
I just decided on something radical: I'm not going to add Twitter and Facebook. I also won't add Reddit support. I'm still going to research if adding Mastodon support is feasible and I'm adding Apple, Discord and Twitch support if it is easy to do so.

The process of creating an app in the developer portal of Facebook was so complicated I just kind of gave up. I already added Twitter support (see [commit d3d244](https://github.com/Friedenspanzer/meeple-party/commit/d3d424477a97faf5bc4770a597c5bf7126c652c5)) but I'm going to remove it. After thinking about this for a very long time I decided I don't have to support corporations I don't like. There's a reason I have created a Mastodon account for Meeple Party instead of a Twitter account and I'm contemplating deleting the subreddit I created.

Yes, Apple, Discord and Twitch aren't _great_, but I dont' _dis_like any of them either. And it will make onboarding for new users a bit harder if they can't log in through their favourite service (chances are that would have been facebook for a whole lot of people). But as this is not a commercial project and my lifelihood doesn't depend on attracting as many users as possible I'm just going through with this.
